lib/content-services/src/lib/content-node-selector/content-node-selector.component.spec.ts

```diff
@@ -393,4 +393,32 @@ describe('ContentNodeSelectorComponent', () => {
             expect(tabGroup).toBe(undefined);
         });
     });
+
+    describe('Breadcrumb', () => {
+        let contentNodePanel;
+
+        beforeEach(() => {
+            contentNodePanel = fixture.debugElement.query(By.directive(ContentNodeSelectorPanelComponent));
+            spyOn(component, 'onBreadcrumbFolderTitleEvent');
+        });
+
+        it('should emit a breadcrumbFolderTitleEvent when the folder is changed', () => {
+            const fakeNodeEntry = new Node({ id: 'fakeId' });
+            const nodeEntryEvent = new NodeEntryEvent(fakeNodeEntry);
+
+            contentNodePanel.componentInstance.onFolderChange(nodeEntryEvent);
+            fixture.detectChanges();
+
+            expect(component.onBreadcrumbFolderTitleEvent).toHaveBeenCalled();
```

Comment:
I suggest you use toHaveBeenCalledWith(something) instead of checking that it was just called